api/src/main/java/org/openmrs/Location.java

```diff
@@ -774,4 +776,20 @@ public String getAddress15() {
 	public void setAddress15(String address15) {
 		this.address15 = address15;
 	}
+	
+	/**
+	 * @return the supportsVisits
+	 * @since 2.4.0
```

Comment:
Isn't this version already released?

Reply:
Hello @dkayiwa, This means that i should alter to version 2.4.1? or 2.5.0?
Which is the next release version.